### PR TITLE
Styles and Drawable assignment

### PR DESCRIPTION
**Pull request by Siddheshwar-Kendre**

### CODE REVIEW

File: StylesAndDrawable/app/src/main/java/com/android/stylesanddrawable/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import android.widget.ImageView;
+
+public class MainActivity extends AppCompatActivity {
+    private static final String COUNTER_STATE="counter_state";
+    private int level=0;
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        ImageView wifi_level=findViewById(R.id.wifi_level_image);
```
Comment (TheAndroidTeacher) [inline StylesAndDrawable/app/src/main/java/com/android/stylesanddrawable/MainActivity.java:18]: Use camel case for variable name in java/kt files.
wifiLevel

File: StylesAndDrawable/app/src/main/java/com/android/stylesanddrawable/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        ImageButton plus=findViewById(R.id.plus_button);
+        ImageButton minus=findViewById(R.id.minus_button);
+        ImageButton turnOff=findViewById(R.id.turnoff_button);
+
+        plus.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                if(level==5)
+                    level=0;
```
Comment (TheAndroidTeacher) [inline StylesAndDrawable/app/src/main/java/com/android/stylesanddrawable/MainActivity.java:27]: should wifi_level.setImageLevel(level); be called from here?

File: StylesAndDrawable/app/src/main/res/layout-land/activity_main.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (TheAndroidTeacher) [inline StylesAndDrawable/app/src/main/res/layout-land/activity_main.xml:1]: This layout for landscape file contains no views. Please add the proper layout for the landscape mode.

**Review by TheAndroidTeacher — COMMENTED** [review]
Added few comments

File: StylesAndDrawable/app/src/main/res/layout-land/activity_main.xml (hunk 3 above)
Comment (Siddheshwar-Kendre) [reply StylesAndDrawable/app/src/main/res/layout-land/activity_main.xml:1]: Okay i will update it today.

**Pull request merged**